views/pages/ChartsPage.jsx

```diff
@@ -1,80 +1,26 @@
-import React, { useState, useEffect } from 'react'
+import React from 'react'
 import { Box, Button } from '@mui/material'
 import { Link, useOutletContext } from 'react-router-dom'
 
-import api from '../api'
 import ChartsTable from '../tables/ChartsTable'
 import PageHeader from '../components/PageHeader'
 import ShareChart from '../components/ShareCharts'
 import { routes } from '../routes/routes'
 
```

Comment:
Nit, but this line can be removed so it's all one block

Reply:
[f69e1dd](https://github.com/AMP-SCZ/dpdash/pull/500/commits/f69e1dd2e2176c10c6ff393ebef0663883ac1cdd)